src/main/java/com/google/sps/servlets/GmailActionableEmailsServlet.java

```diff
@@ -0,0 +1,133 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.api.client.auth.oauth2.Credential;
+import com.google.api.services.gmail.model.Message;
+import com.google.sps.model.AuthenticatedHttpServlet;
+import com.google.sps.model.AuthenticationVerifier;
+import com.google.sps.model.GmailClient;
+import com.google.sps.model.GmailClientFactory;
+import com.google.sps.model.GmailClientImpl;
+import com.google.sps.utility.JsonUtility;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Servlet to get actionable emails from user's gmail account, as defined by specific words in an
+ * email's subject line. Used by "Assign" panel on client.
+ */
+@WebServlet("/gmail-actionable-emails")
```

Comment:
I am not well versed on the design you decided on for the Assign panel - would it be easier if you had a single Assign servlet and have GmailActionableEmails as a client?

Reply:
I think maybe this is a naming issue - this is the only servlet used for getting the actionable emails from gmail (and the only servlet used for getting information from the panel). 

The other servlet used will be tasks, since I will be posting tasks when appropriate. I was thinking of grouping them together, but I don't think it makes sense semantically (posting tasks and getting actionable emails is two different functionalities - the server design should not be dictated by the client's use of the servlets). 

Just my two cents - let me know if this cleared it up, or if you think we should switch some names up on the servlets for clarity.

Reply:
Yeah, I agree that it's probably the names that are a little confusing (same issue with my servlets and clients). I think it's fine to leave it for now, we can always refactor later when it's much more obvious if something needs to be renamed.

Reply:
RESOLVED